src/parser/xasl_generation.c

```diff
@@ -7029,6 +7029,10 @@ pt_make_regu_subquery (PARSER_CONTEXT * parser, XASL_NODE * xasl, const UNBOX un
 	      PT_ERRORm (parser, node, MSGCAT_SET_PARSER_SEMANTIC, MSGCAT_SEMANTIC_OUT_OF_MEMORY);
 	      regu = NULL;
 	    }
+	  if (pt_prepare_corr_subquery_hash_result_cache (parser, (PT_NODE *) node, xasl))
```

Comment:
빈 줄로 구분을 해주시면 좋을 것 같습니다.

```suggestion

	  if (pt_prepare_corr_subquery_hash_result_cache (parser, (PT_NODE *) node, xasl))
```